### PR TITLE
cmio-display updates

### PR DESCRIPTION
**Pull request by 6by9**
Tweaks based on questions asked at https://www.raspberrypi.org/forums/viewtopic.php?f=98&t=311167

### CODE REVIEW

File: hardware/computemodule/cmio-display.md [hunk 1]
```diff
@@ -95,8 +95,12 @@ DISPLAY_DSI_PORT
 The I2C port, SDA and SCL pin numbers are self explanatory. DISPLAY_DSI_PORT
 selects between DSI1 (the default) and DSI0.
 
+Once all the required changes have been made to the ```dts``` file, it needs to be compiled and placed on the boot partition of the device.
+
+Instructions for doing this can be found on the [Pin Configuration](https://github.com/raspberrypi/documentation/blob/master/configuration/pin-configuration.md) page.
+
```
Comment (JamesH65) [inline hardware/computemodule/cmio-display.md:101]: this needs to be a relative link so that it works both in github and on the docs site.

Comment (6by9) [reply hardware/computemodule/cmio-display.md:101]: Doh. Sorry.

**Comment by 6by9** [comment]
Updated.

File: hardware/computemodule/cmio-display.md [hunk 2]
```diff
@@ -95,8 +95,12 @@ DISPLAY_DSI_PORT
 The I2C port, SDA and SCL pin numbers are self explanatory. DISPLAY_DSI_PORT
 selects between DSI1 (the default) and DSI0.
 
+Once all the required changes have been made to the ```dts``` file, it needs to be compiled and placed on the boot partition of the device.
```
[the remark this answers is not in the document]
Comment (lurch) [reply hardware/computemodule/cmio-display.md:98]: (I'll ignore this for now, as I see there are also several other places in this file where triple-backticks are used, when only single-backticks would be needed!)

**Pull request merged**